nixos/modules/services/torrent/transmission.nix

```diff
@@ -492,6 +503,10 @@ in
         # https://gitlab.com/apparmor/apparmor/-/wikis/AppArmorStacking#seccomp-and-no_new_privs
         px ${cfg.settings.script-torrent-done-filename} -> &@{dirs},
       ''}
+
+      ${optionalString (cfg.webHome != null) ''
+        r ${cfg.webHome}/**,
+      ''}
```

Comment:
This is great. Regarding the `extraAppArmorPaths` option added in https://github.com/NixOS/nixpkgs/pull/259663 , I'm not familiar with using AppArmor, so I'll ask: In general with transmission, if these AppArmor paths are considered only when the daemon runs - in the NixOS service, perhaps we should add an `extraAppArmorPaths` option to the NixOS service instead of the package?

The guiding principle is that overriding a package is more expensive in terms of building resources in comparison to changing options in the service. If the above suggestion would be acceptable for you, I'd also revert 71ab0b6c1c94fdc5d702bbe401bbf841f0d2257b in the same commit.

Reply:
I added `extraAppArmorPaths` to the derivation in order to support `flood-for-transmission` with AppArmor enabled. I think it's reasonable to revert it there; I'm not sure if it's worth porting it over to the module if there's no other use for it.

Reply:
> I added `extraAppArmorPaths` to the derivation in order to support `flood-for-transmission` with AppArmor enabled. I think it's reasonable to revert it there; I'm not sure if it's worth porting it over to the module if there's no other use for it.

I think that reverting that commit in this PR, in a separate commit is most appropriate - since the `webHome` committed here is a better solution to the AppArmor issue you fixed there.

I also think that there could be other usages for additional AppArmor paths, but that could be left to another PR or for another user to find the need for it.

Reply:
Sounds reasonable, I reverted it in a separate commit.